schemas/context/direct-marketing.schema.json

```diff
@@ -60,6 +60,21 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description":
             "Offer in a marketing message (email, SMS, Push Notifications etc) has been opened/swyped/touched/interacted with by the recipient."
+        },
+        "xdm:impressions": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description":
+            "The direct-marketing message ( Push Notifications, In-App etc) has been seen by the end user."
+        },
+        "xdm:inAppClicks": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description":
+            "Click(s) actions on an In-App message."
```

Comment:
Does this need its own clicks metric? There is already a `clicks` metric at 
https://github.com/adobe/xdm/blob/99643dace9e262773327dd570fdfd14561d416ce/schemas/context/direct-marketing.schema.json#L22

Reply:
For InApp channel we calculated InAppClickThroughRate as ( unique InAppClicks ) / ( Impressions ).
In addition to this new clickThroughRate formula we also show inAppClick metric on the workspace. So this is required to distinguish between email clicks and inApp clicks.

Reply:
Going purely by the xdm(so as to not be influenced by application logic), is it not enough to send `channel=inApp` and `metric=clicks` to infer what you need?

Reply:
Yes that should be enough